.github/workflows/test.yml

```
@@ -55,3 +55,27 @@ jobs:
 
       - name: Test
         run: yarn test
+
+  test-ui:
+    name: Test UI
+    runs-on: ubuntu-latest
```

UI tests can get stuck sometimes. so it is best to add a timeout:

```suggestion
    timeout-minutes: 10
    runs-on: ubuntu-latest
```